src/Core/maps/src/Handlers/Map/MapHandler.Android.cs

```diff
@@ -71,7 +71,13 @@ protected override void DisconnectHandler(MapView platformView)
 
 		protected override MapView CreatePlatformView()
 		{
-			MapView mapView = new MapView(Context);
+			GoogleMapOptions options = new GoogleMapOptions();
```

Comment:
@PureWeen I wonder if this is an example of something we should be using commands for - or maybe an attached property? Like a platform specific attached property?